src/azure/basic-azure-cli-facade.ts

```diff
@@ -121,7 +166,7 @@ export class BasicAzureCliFacade implements AzureCliFacade {
     }
     if (result.stderr.includes("az login")) {
       log.info(
-        `You are not logged in into Azure CLI. Please disconnect from azure with "${CLICommand} config --disconnect Azure" or login into Azure CLI.`,
+        `You are not logged in into Azure CLI. Please disconnect from azure with "${CLIName} config --disconnect Azure" or login into Azure CLI.`,
```

Comment:
When dealing with commands, always use CLICommand as it is the lowercase variant.

Reply:
changed